### DISCUSSION TITLE
Can't change `FontFamily`

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#13584] [Q&A]
```
<TextBlock
    FontFamily="{Binding Font}"
    FontSize="{Binding Size}"
    Foreground="{Binding FontColor}"
    Text="{Binding Content}"
    TextElement.FontFamily="{Binding Font}" />
```

```
public FontFamily Font
{
    get
    {
        return ElementInfo.Font ?? FontFamily.DefaultFontFamilyName;
    }
    set
    {
        ElementInfo.Font = value.Name;
        OnPropertyChanged(nameof(Font));
        GetNewMargin();
    }
}
```

but use may type is same.

![](https://github.com/AvaloniaUI/Avalonia/assets/26276037/851a30bb-70fd-4f64-aa61-9c0715780754)
![](https://github.com/AvaloniaUI/Avalonia/assets/26276037/f322d93c-eb4a-4b2f-815f-d334c3f0475a)

**Comment by Coloryr** [comment]
this font list read from system fonts.
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/c05dd0d4-fa00-4b01-9c43-e16e6d45cc57)

**Comment by Coloryr** [comment]
work in `11.0.5`
no work in `11.1.999-cibuild0041722-beta`

**Reply by Coloryr** [reply]
https://github.com/AvaloniaUI/Avalonia/commit/da523ffcaff11d3ad50fc7faf57349d634f96999
this start no work

**Reply by maxkatz6** [reply]
This PR has zero changes related to fonts. Are you sure?

**Reply by Coloryr** [reply]
https://github.com/AvaloniaUI/Avalonia/issues/13585#issuecomment-1808112464

**Reply by Coloryr** [reply]
https://github.com/AvaloniaUI/Avalonia/issues/13585#issuecomment-1808112464

**Comment by maxkatz6** [comment]
cc @Gillibald
